homeassistant/components/lutron/config_flow.py

```diff
@@ -0,0 +1,107 @@
+"""Config flow to configure the Lutron integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+from urllib.error import HTTPError
+
+from pylutron import Lutron
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class LutronConfigFlow(ConfigFlow, domain=DOMAIN):
+    """User prompt for Main Repeater configuration information."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """First step in the config flow."""
+
+        # Check if a configuration entry already exists
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
+
+        errors = {}
+
+        if user_input is not None:
+            ip_address = user_input[CONF_HOST]
```

Comment:
We need a way for the user to change this, can we add it already in this PR or do you want to add an option flow in a follow-up?